app/domain/util/stubs/deep_double.rb

```diff
@@ -0,0 +1,72 @@
+module Util
+  module Stubs
+    class DeepDouble
+
+      class LookupReturnValue
```

Comment:
How do you feel about splitting `LookupReturnValue` into its own file? It took me a minute to tease apart `DeepDouble` from `LookupReturnValue`, and I imagine others might have the same problem.

Ref: https://github.com/rubocop-hq/ruby-style-guide#file-classes